src/script.js

```diff
@@ -197,6 +197,17 @@
 			document.body.appendChild(a);
 			a.click();
 			a.remove();
+		};
+
+		saveHtmlBtn.addEventListener('click', function (e) {
+			save_file();
+		});
+		
+		window.addEventListener('keydown',function(event) {
+			if ((event.ctrlKey)&&(event.keyCode==83)){
+				event.returnValue=false;
```

Comment:
I just find Chrome has use `event.preventDefault()` to cancel the default key down event